braid/_braid.h

```diff
@@ -369,10 +406,12 @@ typedef struct _braid_Core_struct
    braid_Real    old_fine_tolx;    /**< Allows for storing the previously used fine tolerance from GetSpatialAccuracy */
    braid_Int     tight_fine_tolx;  /**< Boolean, indicating whether the tightest fine tolx has been used, condition for halting */
    braid_Int     rfactor;          /**< if set by user, allows for subdivision of this interval for better time accuracy */
+   braid_Basis   lvectors;         /**< if Delta correction is set, contains reference to the tangent vectors to be propagated across the c-interval */
```

Comment:
maybe this should be declared up above with ua, va, and fa?  Seems maybe more relevant there?

Reply:
This is here because it is only used in the context of StepStatus. This is how Lyapunov vectors are passed to the user's step function, and is only a reference, so there is no actual allocation for it either. I don't really have a problem with moving it up if you'd like.

Reply:
Oh, I see, maybe the comment could end with something like "..across the c-interval, for giving user access to vectors in Status routines"

I think that it's in the right place, since it deals with Status